packages/dds/sequence/src/sharedString.ts

```diff
@@ -238,6 +239,22 @@ export class SharedString
 		return this.client.findTile(startPos ?? 0, tileLabel, preceding);
 	}
 
+	/**
+	 * Searches a string for the nearest marker in either direction to a given start position.
+	 * The search will include the start position, so markers at the start position are valid
+	 * results of the search.
+	 * @param startPos - Position at which to start the search
+	 * @param markerLabel - Label of the marker to search for
+	 * @param forwards - Whether the desired marker comes before (false) or after (true) `startPos`
+	 */
+	public searchForMarker(
+		startPos: number | undefined,
```

Comment:
can we just make this required? I don't think it's even particularly convenient for a user of this to pass `undefined`, and it's certainly not 'obvious' what it should do.